UIKitDZ/UIKitDZ/View/FirstViewCell.swift

```diff
@@ -0,0 +1,217 @@
+
+    private let pageControl: UIPageControl = {
+        let pageControl = UIPageControl()
+        pageControl.numberOfPages = 3
+        pageControl.tintColor = .black
+        pageControl.backgroundStyle = .minimal
+        return pageControl
+    }()
+
+    // MARK: Стеквью ячейки добавляется в стек
+
+    lazy var stackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.alignment = .center
+        return stackView
+    }()
+
+    // MARK: Установка ячейки
+
+    func setupCell(post: Other) {
+        scrollView.addSubview(stackView)
+        contentView.addSubview(scrollView)
+        contentView.addSubview(pageControl)
+
+        scrollView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        pageControl.translatesAutoresizingMaskIntoConstraints = false
+
+        NSLayoutConstraint.activate([
+            scrollView.topAnchor.constraint(equalTo: contentView.topAnchor),
+            scrollView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor),
+            scrollView.trailingAnchor.constraint(equalTo: contentView.trailingAnchor),
+            scrollView.bottomAnchor.constraint(equalTo: contentView.bottomAnchor),
+            scrollView.heightAnchor.constraint(equalToConstant: 411),
+
+            stackView.topAnchor.constraint(equalTo: scrollView.topAnchor),
+            stackView.leadingAnchor.constraint(equalTo: scrollView.leadingAnchor),
+            stackView.trailingAnchor.constraint(equalTo: scrollView.trailingAnchor),
+            stackView.bottomAnchor.constraint(equalTo: scrollView.bottomAnchor),
+            stackView.heightAnchor.constraint(equalTo: scrollView.heightAnchor),
+
+            pageControl.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -90),
+            pageControl.centerXAnchor.constraint(equalTo: scrollView.centerXAnchor),
+            pageControl.heightAnchor.constraint(equalToConstant: 10),
+        ])
+
+        renderUI(post: post)
+    }
+
+    // MARK: Обрабатываем визуал
+
+    private func renderUI(post: Other) {
+        let view = UIView()
+        stackView.addSubview(view)
+        view.translatesAutoresizingMaskIntoConstraints = false
+
+        postImages = [post.photo, post.photo, post.photo]
+
+        let imageView = makeImage(imageName: post.photo, in: view)
+
+        let avatarImage = makeImage(imageName: post.friendPhoto, in: view)
+
+        let label = UILabel()
+        label.font = UIFont(name: "Verdana", size: 12)
+        label.text = post.name
+        view.addSubview(label)
+        label.translatesAutoresizingMaskIntoConstraints = false
+
+        let likeButton = makeButton(imageName: "like", in: view)
+
+        let commentButton = makeButton(imageName: "comment", in: view)
+
+        let shareButton = makeButton(imageName: "send", in: view)
+        let bookmarkButton = makeButton(imageName: "bookmark", in: view)
```

Comment:
```suggestion
        let likeButton = makeButton(imageName: "like", in: view)
        let commentButton = makeButton(imageName: "comment", in: view)
        let shareButton = makeButton(imageName: "send", in: view)
        let bookmarkButton = makeButton(imageName: "bookmark", in: view)
```
Вынести названия изображений в константы

Reply:
осознаю